From: Carlos Maddela <e7appew@gmail.com>
Date: Tue, 1 Nov 2016 13:46:44 +1100
Subject: Prevent chrome-devtools LICENSE from being installed in undesired
 location.

Description: Prevent chrome-devtools LICENSE from being installed in undesired location.
Author: Carlos Maddela <e7appew@gmail.com>
Origin: vendor
Forwarded: not-needed
Last-Update: 2016-11-01
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
---
 .../web/static/images/chrome-devtools/LICENSE      | 30 ----------------------
 1 file changed, 30 deletions(-)
 delete mode 100644 mitmproxy/web/static/images/chrome-devtools/LICENSE

diff --git a/mitmproxy/web/static/images/chrome-devtools/LICENSE b/mitmproxy/web/static/images/chrome-devtools/LICENSE
deleted file mode 100644
index 6e4f8b9..0000000
--- a/mitmproxy/web/static/images/chrome-devtools/LICENSE
+++ /dev/null
@@ -1,30 +0,0 @@
-// Copyright 2014 The Chromium Authors. All rights reserved.
-//
-// The Chromium Authors can be found at
-// http://src.chromium.org/svn/trunk/src/AUTHORS
-//
-// Redistribution and use in source and binary forms, with or without
-// modification, are permitted provided that the following conditions are
-// met:
-//
-//    * Redistributions of source code must retain the above copyright
-// notice, this list of conditions and the following disclaimer.
-//    * Redistributions in binary form must reproduce the above
-// copyright notice, this list of conditions and the following disclaimer
-// in the documentation and/or other materials provided with the
-// distribution.
-//    * Neither the name of Google Inc. nor the names of its
-// contributors may be used to endorse or promote products derived from
-// this software without specific prior written permission.
-//
-// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
-// "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
-// LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
-// A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
-// OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
-// SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
-// LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
-// DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
-// THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
-// (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
-// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
\ No newline at end of file
